### DISCUSSION TITLE
Is it possible to Keep only one Instance of ImageGlass?

### DISCUSSION THREAD

**Discussion by nutnutwin** [d2phap/ImageGlass#900] [Q&A] [answered]
Hi, Thank you for your great work

I have one image opened in ImageGlass, and when I double click another image, 
another ImageGlass instance pops up. 

Wonder if there is anyway I can keep the instance to one? 
I went through the settings and found nothing. Sorry if I missed anything. 

--- ✄ -----------------------
I used to use FastStone, which has one instance by default, 
but ImageGlass is SO MUCH faster, and prettier

Thank you

**Comment by d2phap** [comment, marked as answer]
Hi @nutnutwin 
Yes, you can find it in Settings:
![image](https://user-images.githubusercontent.com/3154213/97796810-999c0600-1c51-11eb-851d-d6b82dec55d5.png)

**Comment by nutnutwin** [comment]
Shame...
How can I miss that,
Thank you!!!
